val_moveit_planner_executor/ModifyAllowedCollision[] modify_collisions

---

bool success
